ddtrace/tracer/option.go

```diff
@@ -118,8 +118,8 @@ type config struct {
 	// statsd is used for tracking metrics associated with the runtime and the tracer.
 	statsd statsdClient
 
-	// samplingRules contains user-defined rules determine the sampling rate to apply
-	// to spans.
+	// samplingRules contains user-defined rules to determine the sampling rate to apply
+	// to spans. Rules might be of a type SamplingRuleSingleSpan or SamplingRuleTrace.
```

Comment:
I think that this added text is simply a detail of the sampling rule and there is no need to add it here.